src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -236,6 +248,12 @@ ChipError::StorageType pychip_DeviceController_StackInit()
 
     factoryParams.enableServerInteractions = true;
 
+    // Hack needed due to the fact that DnsSd server uses the CommissionableDataProvider even
+    // when never starting operational advertising. This will not be used but prevents
```

Comment:
```suggestion
    // when never starting commissionable advertising. This will not be used but prevents
```
The reason it needs it is that it uses it to know whether to start commissionable advertising....